keps/sig-scheduling/5007-device-attach-before-pod-scheduled/README.md

```diff
@@ -1102,6 +1151,12 @@ Think about adding additional work or introducing new steps in between
 [existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
 -->
 
+Yes, when this feature is enabled and if devices corresponding to
+BindingConditions exists in the k8s cluster, there is a potential impact on the
+time it takes for a Pod to transition from creation to a Running state, as well
+as on Pod creation throughput. This is due to waiting for status updates from
+external controllers in the scheduler's PreBind phase.
```

Comment:
I wonder if we can annotate the metrics so people can still see the old version of the metric, excluding any requests that required binding. Can you look into that?